src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -64,16 +57,31 @@ const Body = BaseFormWithPolling.extend(
         this.doLoopback(deviceChallenge.domain, deviceChallenge.ports, deviceChallenge.challengeRequest);
         break;
       case Enums.CUSTOM_URI_CHALLENGE:
-        this.title = loc('oktaVerify.button', 'login');
+        this.title = loc('customUri.title', 'login');
         this.add(View.extend({
           className: 'skinny-content',
           template: hbs`
             <p>
               {{{i18n code="customUri.required.content.p1" bundle="login"}}}
```

Comment:
Now that there is no HTML in this properties value - please remove the triple curly brace: 

```suggestion
              {{i18n code="customUri.required.content.p1" bundle="login"}}
```